src/js_realm.hpp

```diff
@@ -220,6 +223,7 @@ class RealmClass : public ClassDefinition<T, SharedRealm, ObservableClass<T>> {
         {"deleteAll", wrap<delete_all>},
         {"write", wrap<write>},
         {"addListener", wrap<add_listener>},
+        {"wait", wrap<wait_for_download_completion>},
```

Comment:
I get that this is not the real api, but "wait" seems a bit generic to me. Is there any reason it's not simply `onRealmSynchronized` or something similar? I mean, it *is* public, right?